packages/core/src/icon/icon.element.ts

```diff
@@ -87,16 +87,6 @@ export class CdsIcon extends LitElement {
   @property({ type: String })
   title: string;
 
-  /**
-   * @deprecated
-   * Takes a directional value (up|down|left|right) that rotates the icon 90° with the
-   * top of the icon pointing in the specified direction.
-   *
-   * Deprecated in 3.0. Use `direction` instead. `dir` will be removed in 4.0!
-   */
-  @property({ type: String })
-  dir: Directions;
```

Comment:
This property is removed in the `element.ts` file but is still in the CSS. Please fix.